### DISCUSSION TITLE
Any plugins anti-grief to use?

### DISCUSSION THREAD

**Discussion by iDan-23** [Pryaxis/TShock#1955] [Q&A] [answered]
I want to have an plugin on my server that gives players the ability to protect their houses from grief. Is there anything I can use?

**Comment by moisterrific** [comment, marked as answer]
I recommend [House Regions](https://github.com/CoderCow/HouseRegions-Plugin) by CoderCow, it’s not updated for the latest version of the game/TShock so dunno if it will work.

**Reply by iDan-23** [reply]
I was thinking about it, but it requires TerrariaAPI and idk how to set it up

**Reply by hakusaro** [reply]
TSAPI is shipped in TShock

**Reply by iDan-23** [reply]
Soo it works if I just copypaste the content from House Regions?

**Reply by moisterrific** [reply]
You need House Regions and Plugin Common Library. Just have those 2 class libraries (.dlls files) in the ServerPlugins folder.

**Reply by iDan-23** [reply]
Alright. Thanks!
